setup.py

```diff
@@ -265,15 +265,26 @@ def compile_KLU():
         "pandas": [
             "pandas>=1.5.0",
         ],
+        # Note: jax and jaxlib must be pinned to a specific version
+        # to avoid upstream breaking changes.
```

Comment:
I would suggest keeping the pin. Keeping no bounds at all would break things on a future 0.5.X release of `jaxlib`, plus the current GPU support with the Windows community wheels and the macOS Metal plugin is still going to lag behind the current 0.4 releases, so we would want to stay within >=0.4, <0.5 borders so as to not promote different API behaviour for any Jax-related functions.

Reply:
I was not suggesting that we don't pin. Rather we don't need a comment explaining that pinning avoids breaking the build. I like reducing comments whenever possible because in most projects people forget to update them.